package.json

```diff
@@ -5,6 +5,7 @@
   "license": "MIT",
   "dependencies": {
     "@babel/core": "^7.7.7",
+    "@babel/plugin-transform-react-jsx-development": "^7.9.0",
```

Comment:
Do we need this?

Reply:
We're not using jsx anymore, so I don't see why we would.

Reply:
We do. I'm not sure why but after adding sass files and node-sass there is an error during compilation asking for this plugin. Luckily this will all go away once we go back to un-ejected CRA.

Reply:
Sounds good I guess we'll just leave it in as an intermediary step.  Everything else looks good to me!